src/Svg.Model/SvgExtensions.Painting.cs

```diff
@@ -1022,7 +1023,8 @@ internal static bool IsValidStroke(SvgElement svgElement, SKRect skBounds)
         };
 
         var server = svgVisualElement.Stroke;
-        var opacity = AdjustSvgOpacity(svgVisualElement.StrokeOpacity);
+        var opacity = AdjustSvgOpacity(svgVisualElement.Opacity);
+        opacity *= AdjustSvgOpacity(svgVisualElement.StrokeOpacity);
```

Comment:
This breaks stuff

![image](https://user-images.githubusercontent.com/2297442/186350772-8e829fab-f70f-4ac1-8d26-bc67e4f44d3e.png)

Reply:
How do I view the same window?

Reply:
Run https://github.com/wieslawsoltes/Svg.Skia/tree/master/samples/SvgToPng (Windows only)

Reply:
@wieslawsoltes It initializes empty. It seems to need to load a JSON file somewhere using "Load..."?

Reply:
@Happypig375 
https://user-images.githubusercontent.com/2297442/186384239-425c79b3-600f-4d82-8dee-c0d06ca87821.mp4